modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,50 @@
+= Footnotes plugin
+:navtitle: Footnotes
+:description: Insert footnotes into the {productname} editor
+:keywords: footnotes, footnote, insertfootnotes, insert, footnotesupdate, mceInsertFootnote, mceUpdateFootnotes
+:pluginname: Footnotes
+:plugincode: footnotes
+:pluginminimumplan: tierthree
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a footnote into the {productname} editor.
+
+Invoking the plugin adds a superscripted footnote number at the current insertion point, or at the end of the current selection.
```

Comment:
Invoking the plugin sort of makes it sound like that's all it can do. It's just one capability of the plugin. Or is this how we normally write it?

Reply:
Re-written in response to both comments.